tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/NpxHelper.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Runtime.InteropServices;
+
+namespace Azure.Sdk.Tools.Cli.Helpers
+{
+    public interface INpxHelper
+    {
+        public ProcessResult RunNpx(List<string> args, string workingDirectory);
+    }
+
+    public class NpxHelper(IProcessHelper processHelper) : INpxHelper
+    {
+        public ProcessResult RunNpx(List<string> args, string workingDirectory)
+        {
+            var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+            if (isWindows)
+            {
+                return processHelper.RunProcess(
+                    "cmd.exe",
```

Comment:
Is the cmd call necessary?

Reply:
It shouldn't be. I'd prefer that we don't include it if we can.

Reply:
This is because `npx` isn't an exe on windows - it's either a `.cmd` or `.ps1`. Trying to invoke it directly or as `npx.cmd/npx.ps1` doesn't work. Only ways I've found to get it to work on Windows is to either call `cmd.exe /C` or `powershell.exe -Command`.

I mostly based this on an existing example: https://github.com/Azure/azure-sdk-tools/blob/c2862713609a6dbe51aa65f555ffe07832bb3707/tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/SpecValidationTool/SpecValidationTool.cs#L99-L104